integration-tests/run

```diff
@@ -17,7 +17,7 @@
 # for more information about the Cockpit integration test machinery.
 
 ifeq ($(TEST_OS),)
-TEST_OS = rhel-8-3
+TEST_OS = rhel-8-5
```

Comment:
I'd not change this as part of the changes, as it might conflict when backported later on.